### PR TITLE
Lint Helm Charts

### PR DESCRIPTION
**Pull request by svenmueller**
Lint Helm Charts automatically.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+language: python
+python:
+  - "3.6.4"
+  
+jobs:
+  include:
+    - stage: install
+      script: test/travis/install.sh
+    - stage: lint
+      script: test/travis/lint.sh
```
Comment (anas-aso) [inline .travis.yml:12]: @svenmueller why are we using travis ? I thought we agreed on using Cloudbuilder for everything ops related or ?

Comment (svenmueller) [reply .travis.yml:12]: @anas-aso since this repo is public, we started using travis-ci.org for building it. so CI builds/logs are accessible.

**Review by andrii-kovalenko-ct — APPROVED** [review]
Any examples how to apply it on Travis build?

**Comment by svenmueller** [comment]
@andrii-kovalenko-ct travis build will be triggered for every push from a branch or when creating a PR

**Comment by andrii-kovalenko-ct** [comment]
@svenmueller Travis build is triggered independently from k8s and helm setup. My question is what should i use/call in `.travis.yml` when travis build is triggered

**Comment by svenmueller** [comment]
@andrii-kovalenko-ct not sure what you mean. You don't need to adjust the `.travis.yml`in this repo for a specific chart. Maybe i don't get your point yet :)

**Pull request merged**